### PR TITLE
US174-TA175: Switch AccountController

### PR DESCRIPTION
**Pull request by evgenii-nikulin**
- Switch api call from `AccountControlle` to `AccountsControlle`
- Changed email validation backend logic
- Fixed prettier config

### CODE REVIEW

File: src/.prettierrc [hunk 1]
```diff
@@ -1,8 +1,9 @@
-# .prettierrc
```
Comment (yuya-yuliya) [inline src/.prettierrc:1]: Why this file changed?

File: src/Feature/Account/client/Integration/api/Account/api.ts [hunk 2]
```diff
@@ -1,132 +1,128 @@
...
+): Promise<Result<Commerce.ValidateEmail>> => {
+  return axios
+    .post<EmailValidationResponse>(`${routeBase}/validate`, validateEmailRequest)
+    .then((response) => {
+      return { data: { invalid: false, inUse: response.data.data.inUse } };
+    })
+    .catch((error: AxiosError) => {
+      if (error.response.status === 400) {
+        return { data: { invalid: true, inUse: false } };
+      }
```
Comment (yuya-yuliya) [inline src/Feature/Account/client/Integration/api/Account/api.ts:54]: Do we have data in response if status code is 400?

File: src/.prettierrc (hunk 1 above)
Comment (evgenii-nikulin) [reply src/.prettierrc:1]: I wrote , prettier didn't work and I fixed that

File: src/Feature/Account/client/Integration/api/Account/api.ts (hunk 2 above)
Comment (evgenii-nikulin) [reply src/Feature/Account/client/Integration/api/Account/api.ts:54]: No, status code 400 is returning when request model not valid.

**Review by yuya-yuliya — APPROVED** [review]

File: src/Feature/Account/client/Integration/Account/sagas.ts [hunk 3]
```diff
@@ -79,26 +80,19 @@ export function* changePassword(action: Action<ChangePasswordPayload>) {
...
-    changePasswordPayload
-  );
+  const { data, error }: Result<boolean> = yield call(Api.changePassword, payload);
 
   if (error) {
     return yield put(actions.ChangePasswordFailure(error.message, error.stack));
   }
 
-  if (!data.passwordChanged) {
+  if (!data) {
```
Comment (larionovadaria) [inline src/Feature/Account/client/Integration/Account/sagas.ts:95]: what is data now? why `!data.passwordChanged` condition was removed?

File: src/Feature/Account/client/SignUpForm/Component.tsx [hunk 4]
```diff
@@ -78,8 +78,12 @@ export default class SignUpFormComponent extends Jss.SafePureComponent<SignUpPro
                       <i className="fa fa-spinner fa-spin" />
                     </span>
                   )}
-                  {accountValidation.status === LoadingStatus.Loaded &&
-                    accountValidation.invalid && <span className="form-field-error-message">{accountValidation.inUse ? 'Email is already in use!' : 'Email is invalid!'}</span>}
+                  {accountValidation.status === LoadingStatus.Loaded && (
+                    <span className="form-field-error-message">
+                      {accountValidation.inUse && 'Email is already in use!'}
+                      {accountValidation.invalid && 'Email is invalid!'}
```
Comment (larionovadaria) [inline src/Feature/Account/client/SignUpForm/Component.tsx:84]: Add all messages to sitecore dictionary items and make them translatable

**Review by ghost — CHANGES_REQUESTED** [review]
Please see my comments

File: src/Feature/Account/client/Integration/Account/sagas.ts (hunk 3 above)
Comment (evgenii-nikulin) [reply src/Feature/Account/client/Integration/Account/sagas.ts:95]: because I passed `boolean` value in `data` instead object

**Review by ghost — APPROVED** [review]

**Pull request merged**